### ISSUE TITLE
module rgpluginsgm_psvita get unicorn error

### ISSUE THREAD

**Issue by Zangetsu38** [Vita3K/Vita3K#1095]
# Issue
- Module see on gravity duck and Super Box Land Demake, and probably some other, get directly unicorn error on
boot, rename it can get mùore boot, but controller no working

# Log:
- Super Box Land Demake
[vita3k.log](https://github.com/Vita3K/Vita3K/files/5800112/vita3k.log)

- Gravity Duck
[vita3k.log](https://github.com/Vita3K/Vita3K/files/5800113/vita3k.log)

**Comment by Zangetsu38** [comment]
update for this, on recent build i have see some week ago, rename it game get works, and controller now is works, is full playable for all game using this module.
need found how fix boot and how fix load this module

**Comment by nishinji** [comment]
Outdated

**Comment by Zangetsu38** [comment]
Not really, is keep same, actually code using workaround for ignore it...

**Comment by nishinji** [comment]
Please change the title then.
Unicorns don't exist.

**Comment by Zangetsu38** [comment]
is cpu error, is same, but yes
